RecoLocalMuon/Configuration/python/RecoLocalMuon_cff.py

```diff
@@ -49,8 +49,8 @@
 _phase2_muonlocalreco = _run3_muonlocalreco.copy()
 _phase2_muonlocalreco += me0LocalReco
 
-from Configuration.Eras.Modifier_run2_GEM_2017_MCTest_cff import run2_GEM_2017_MCTest
-run2_GEM_2017_MCTest.toReplaceWith( muonlocalreco , _run2_GEM_2017_muonlocalreco )
+from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
```

Comment:
probably should point out in PR title that GEM reco will be enabled in Phase 1 workflows now

also, can `Modifier_run2_GEM_2017_MCTest_cff` be deleted completely now?

Reply:
i'll discuss this in our gem meeting to see what is best for us.

Reply:
BTW, this can not be merged in the release until the GEM packing/unpacking is in place.